### PR TITLE
feat(atomic): exposes numberOfFoldedResults

### PR DESCRIPTION
**Pull request by louis-bompart**
https://coveord.atlassian.net/browse/KIT-2985

### CODE REVIEW

**Comment by louis-bompart** [comment]
SHould prolly do some e2e

File: packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx [hunk 1]
```diff
@@ -98,6 +98,16 @@ export class AtomicFoldedResultList implements InitializableComponent {
    */
   @Prop({reflect: true}) public childField?: string;
 
+  /**
+   * The number of child results to fold under the root collection element, before expansion.
```
Comment (fbeaudoincoveo) [inline packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx:102]:
```suggestion
   * The initial number of child results to request for each folded collection, before expansion.
```

File: packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx [hunk 2]
```diff
@@ -98,6 +98,16 @@ export class AtomicFoldedResultList implements InitializableComponent {
...
+   *
+   * @defaultValue `2`
+   *
+   * @example For an email thread with a total of 20 messages, using the default value of `2` will display the first two messages.
```
Comment (fbeaudoincoveo) [inline packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx:106]:
```suggestion
   * @example For an email thread with a total of 20 messages, using the default value of `2` will request the top two child messages, based on the current sort criteria and query, to be returned as children of the parent message.
```

File: packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx [hunk 3]
```diff
@@ -98,6 +98,16 @@ export class AtomicFoldedResultList implements InitializableComponent {
...
+   * The user can then click to expand the collection and see the remaining 18 messages, see the `atomic-load-more-children-results` component.
```
Comment (fbeaudoincoveo) [inline packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx:107]:
```suggestion
   * The user can then click to expand the collection and see the remaining messages that match the current query (i.e., not necessarily all remaining 18 messages). Those messages will be sorted based on the current sort criteria (i.e., not necessarily by date). See the `atomic-load-more-children-results` component.
```

File: packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (fbeaudoincoveo) [reply packages/atomic/src/components/search/result-lists/atomic-folded-result-list/atomic-folded-result-list.tsx:106]: I made a suggestion to try to summarize the behavior in the example, but linking to the docs might also be a good idea 😅

**Review by fbeaudoincoveo — APPROVED** [review]
Were there any customer requests to expose this property in Atomic?

**Comment by louis-bompart** [comment]
> Were there any customer requests to expose this property in Atomic?

Yep 👍

**Pull request merged**